pkg/apis/core/validation/seed.go

```diff
@@ -270,6 +271,17 @@ func validateSeedNetworks(seedNetworks core.SeedNetworks, fldPath *field.Path, i
 	if seedNetworks.Nodes != nil {
 		networks = append(networks, cidrvalidation.NewCIDR(*seedNetworks.Nodes, fldPath.Child("nodes")))
 	}
+	if seedNetworks.VPN != nil {
+		vpnNetwork = *seedNetworks.VPN
+	} else {
+		if core.IsIPv6SingleStack(seedNetworks.IPFamilies) {
```

Comment:
Shouldn't the default IP range depend on `primaryIPFamily` instead?
That's at least what's checked a few lines below
`allErrs = append(allErrs, cidrvalidation.ValidateCIDRIPFamily(networks, string(primaryIPFamily))...)`